specification.md

```diff
@@ -3215,17 +3215,18 @@ end: true
 
 Parallel state defines a collection of `branches` that are executed in parallel.
 A parallel state can be seen a state which splits up the current workflow instance execution path
-into multiple ones, one for each of each branch. These execution paths are performed in parallel
+into multiple ones, one for each branch. These execution paths are performed in parallel
 and are joined back into the current execution path depending on the defined `completionType` parameter value.
 
 The "completionType" enum specifies the different ways of completing branch execution:
-* and: All branches must complete execution before state can perform its transition. This is the default value in case this parameter is not defined in the parallel state definition.
-* xor: State can transition when one of the branches completes execution
-* n_of_m: State can transition once `n` number of branches have completed execution. In this case you should also
-specify the `n` property to define this number.
+* allOf: All branches must complete execution before state can perform its transition. This is the default value in case this parameter is not defined in the parallel state definition.
```

Comment:
the state can also define "end" to end workflow execution, not just transition. should probably add